dev-infra/commit-message/validate.spec.ts

```diff
@@ -31,44 +31,34 @@ const SCOPES = config.commitMessage.scopes.join(', ');
 const INVALID = false;
 const VALID = true;
 
+function expectValidationResult(predicate: any, valid: boolean, errors: string[] = []) {
```

Comment:
Predicate is typically a boolean function, so it sounds a little misleading here. How about:

```suggestion
function expectValidationResult(validationResult: any, valid: boolean, errors: string[] = []) {
```